docs/cloud-native-security/cspm-get-started.asciidoc

```diff
@@ -40,24 +40,48 @@ To set up CSPM for AWS, add the CSPM integration and then enable cloud account a
 === Set up cloud account access
 The CSPM integration requires access to AWS’s built-in https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies_job-functions.html#jf_security-auditor[`SecurityAudit` IAM policy] in order to discover and evaluate resources in your cloud account. There are several ways to provide access:
 
-* <<cspm-use-instance-role,Default instance role>>
+* <<cspm-use-instance-role,Default instance role (recommended)>>
 * <<cspm-use-keys-directly,Direct access keys>>
 * <<cspm-use-temp-credentials,Temporary security credentials>>
 * <<cspm-use-a-shared-credentials-file,Shared credentials file>>
 * <<cspm-use-iam-arn, IAM role Amazon Resource Name (ARN)>>
 
-Regardless of which option you use, you’ll need to attach AWS’s built-in https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies_job-functions.html#jf_security-auditor[`SecurityAudit` IAM policy].
+IMPORTANT: Regardless of which option you use, you’ll need to attach AWS’s built-in https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies_job-functions.html#jf_security-auditor[`SecurityAudit` IAM policy].
 
 [discrete]
 [[cspm-use-instance-role]]
-==== Option 1 - Use default instance role
-Follow AWS's https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/iam-roles-for-amazon-ec2.html[IAM roles for Amazon EC2] documentation to create an IAM role using the IAM console, which automatically generates an instance profile.
-
-During setup, do not fill in any option in the "Setup Access" section. Click *Save and continue*.
+=== Option 1 - Use default instance role (recommended)
+Follow AWS's https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/iam-roles-for-amazon-ec2.html[IAM roles for Amazon EC2] documentation to create an IAM role using the IAM console, which automatically generates an instance profile
+
+*First, create an IAM role:*
+
+. In AWS, go to your IAM dashboard. Click *Roles*, then *Create role*.
+. On the "Trusted entity type" page, select *AWS service*, and *EC2*. Click *Next*.
```

Comment:
Page names should be bold like the other elements the user is interacting with. Repeat for other steps.
```suggestion
. On the *Trusted entity type* page, select *AWS service*, and *EC2*. Click *Next*.
```